### ISSUE TITLE
[Vue warn]: Error in render

### ISSUE THREAD

**Issue by AKopytenko** [livelybone/vue-datepicker#6]
I'm trying to connect the calendar component as described in the README.MD
When I click on the input field I see an error in console:
```
[Vue warn]: Error in render: "TypeError: Cannot convert undefined or null to object"
...
on: {
      "click": function click($event) {
        **_vm.canEdit ? _vm.hide(false) : '';**
      },
...
```
The calendar doesn't open.

That's my code:
```
<template>
    <div class="datapicker">
        <Datepicker
            id="DTPicker"
            value="2020-05-07"
            min="2020-05-06" 
            max="2020-05-07" 
            :dayStr="localDaysName"
        />
    </div>
</template>

<script>
import { Datepicker } from '@livelybone/vue-datepicker';

export default {
    name: 'DTPicker',
    data() {
        return {
            localDaysName: ['Пн', 'Вт', 'Ср', 'Чт', 'Пт', 'Сб', 'Вс']
        }
    },
    components: {
        Datepicker
    }
}
</script>

<style lang="scss">
@import url('../../node_modules/@livelybone/vue-datepicker/lib/css/index.scss');
</style>
```

**Comment by scramatte** [comment]
Same issue here

**Comment by livelybone** [comment]
This bug has fixed in version 3.1.1. It caused by the undefined popperProps

**Comment by AKopytenko** [comment]
Ok, it's work. Thanks!
